extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -177,8 +180,31 @@ oppia.factory('ItemSelectionInputValidationService', [
                       'rule must include at least 2 options.')
                   });
                 }
+              } else if (rule.type === 'Equals') {
+                if (minAllowedCount > ruleInputs.length) {
+                  warningsList.push({
+                    type: WARNING_TYPES.ERROR,
+                    message: (
+                      'In answer group ' + (answerIndex + 1) + ', ' +
+                      'rule ' + (ruleIndex + 1) + ', the "Equals" rule ' +
+                      'should not have more correct options than minimum ' +
```

Comment:
I think the selected correct options should be >= minAllowedCount and <= maxAllowedCount. You need to take care of both the cases here. Here, the error message is wrong.

Reply:
Yeah I agree I will change it.

Reply:
Dome